### PR TITLE
Add marked positions

### PR DESCRIPTION
**Pull request by luxiya01**
- Add a dictionary of `marked_positions` to algae config yaml file
- Pull hard-coded buoy definitions from the environment file to `marked_positions`
- ~~Publish parameters from the `world_config_file` to the global namespace when the simulation is launched~~

### CODE REVIEW

File: smarc_stonefish_worlds/launch/stonefish.launch [hunk 1]
```diff
@@ -22,6 +23,9 @@
     <param name="utm_zone" type="int" value="$(arg utm_zone)"/>
     <param name="utm_band" type="string" value="$(arg utm_band)"/> 
 
+    <!-- Put world_config_file params in global namespace -->
+    <rosparam file="$(arg world_config_file)" subst_value="true"/>
```
Comment (nilsbore) [inline smarc_stonefish_worlds/launch/stonefish.launch]: Does it need to be in the global namespace? You will still have access to them in `/stonefish_simulator/[your param]`

Comment (luxiya01) [reply smarc_stonefish_worlds/launch/stonefish.launch]: Hmm... I was thinking that we want to access `/marked_positions` regardless in simulation or in real life...?

Comment (nilsbore) [reply smarc_stonefish_worlds/launch/stonefish.launch]: I'm concerned that this pollutes the global namespace with parameters that might be confusing. E.g. `/environment_file`, `/robot_pos`, `/latitude` or `/longitude` might suggest that they have some particular meaning, whereas in reality, they are just implementation parameters of our stonefish simulation environment.

Comment (luxiya01) [reply smarc_stonefish_worlds/launch/stonefish.launch]: Hmm... That is a good point... I'll remove it, sorry for the messy commits :see_no_evil:

**Pull request merged**